Adyen/Validators/Validator.swift

```diff
@@ -22,3 +22,56 @@ public protocol Validator {
     func maximumLength(for value: String) -> Int
     
 }
+
+/// :nodoc:
+public func || (lhs: Validator, rhs: Validator) -> Validator {
+    ORValidator(lhs: lhs, rhs: rhs)
+}
+
+/// :nodoc:
+public func && (lhs: Validator, rhs: Validator) -> Validator {
+    ANDValidator(lhs: lhs, rhs: rhs)
+}
+
+/// Interface to allow for two validators.
+internal protocol CombinedValidator: Validator {
+    var lhs: Validator { get }
+    var rhs: Validator { get }
+}
+
+extension CombinedValidator {
+    
+    func maximumLength(for value: String) -> Int {
```

Comment:
why do you need this assumption? Validation should simply pass for all involved validators, right?

Reply:
You mean decide them each separately in ANDValidator and ORValidator?